src/content/docs/apm/agents/nodejs-agent/installation-configuration/codestream-integration.mdx

```diff
@@ -0,0 +1,102 @@
+---
+title: New Relic CodeStream integration for Node.js
+tags:
+  - Agents
+  - Nodejs agent
+  - Configuration
+metaDescription: The Node.js agent can be integrated with New Relic CodeStream to provide code level metrics.
+---
+
+Display context-sensitive APM data directly in your IDE by integrating [New Relic CodeStream](/docs/codestream/how-use-codestream/performance-monitoring#code-level) with the Node.js agent. Visualize code-level production telemetry in your editor as your write and review code with this integration.
+
+## Get started [#get-started]
+
+First, [install](/docs/codestream/start-here/install-codestream) the New Relic CodeStream extension into your supported IDE of choice, and log in.
+
+<Callout variant="important">
+  The New Relic CodeStream integration is available in [Node.js agent version 9.7.5 and higher](/docs/release-notes/agent-release-notes/net-release-notes) and is disabled by default. To change this configuration, check out our [documentation](/docs/apm/agents/nodejs-agent/installation-configuration/nodejs-agent-configuration#code_level_metrics).
+</Callout>
+
+## Agent attributes [#attributes]
+
+The Node.js agent reports and attaches the following attributes to most spans produced by [web frameworks](https://docs.newrelic.com/docs/apm/agents/nodejs-agent/getting-started/compatibility-requirements-nodejs-agent/#support) in your application.
+
+<Callout variant="important">
+  In some cases, the agent will directly wrap base framework code that is invoked within your application and will not display within your IDE.
```

Comment:
```suggestion
  In some cases, the agent will directly wrap base framework code that's invoked within your application and won't display within your IDE.
```

It's not clear what won't display within my IDE - is it "the agent won't display" or "the framework code won't display" within your IDE?

Reply:
base framework code. But this was just copied from the same doc i referenced above https://docs.newrelic.com/docs/apm/agents/net-agent/other-features/net-codestream-integration/